main.py

```
@@ -0,0 +1,168 @@
+        chunk = await upload_file.read(chunk_size)
+        if not chunk:
+            break
+        size += len(chunk)
+        if size > MAX_FILE_SIZE:
+            raise HTTPException(
+                status_code=413,
+                detail="이미지 파일은 10MB를 넘을 수 없습니다. "
+            )
+    await upload_file.seek(0)
+#----- 비밀번호 규칙 -------------------------------------------
+password_pattern = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[!@#$%^&*+=-]).{5,20}$"
```

Ruff (0.14.11)

Possible hardcoded password assigned to: "password_pattern"

(S105)